FinniversKit/Sources/Recycling/GridViews/Markets/CompactView/CompactMarketsViewCell.swift

```diff
@@ -124,13 +128,15 @@ class CompactMarketsViewCell: UICollectionViewCell {
 // MARK: - Size calculations
 
 extension CompactMarketsViewCell {
+    
     static let contentInsets = UIEdgeInsets(vertical: .spacingS, horizontal: 12)
     static let titleLabelStyle = Label.Style.captionStrong
     static let cellHeight: CGFloat = 34
     static let itemSpacing = CGFloat.spacingS
     static let externalLinkImageSize = CGSize(width: 12, height: 12)
 
-    static func size(for model: MarketsViewModel) -> CGSize {
+    
+    static func size(isFinn: Bool, for model: MarketsViewModel) -> CGSize {
```

Comment:
this is the compact markets view cell
<img width="391" alt="image" src="https://github.com/finn-no/FinniversKit/assets/23578783/a13ff1c6-efe9-4d28-bc60-b54f03be71c1">
did you intend to change anything here?